alongtheway.txt

```diff
@@ -51,7 +51,7 @@ Optional: if you didn’t already do so on Day 1 you can jot down a timeline of
 
 Reflect: As you read these verses today, try to imagine what it must have been like for Noah and his family to obey God day after day after day with no dramatic intervention or sign of blessing. That sounds a lot like my life too. While I may not have heard the audible voice of God or been assigned to build an ark, there are definitely areas in which I’ve been called to obedience and honestly, it can be hard for me to wait to receive the blessing of my obedience. I’ve fasted from something for a month and it has been eye-opening to see just how immediate-reward-focused I am! I am only on week two of my fast and I’m already itching to be done, questioning if this is actually doing me any good. What must it have been like for Noah as he logged year after year after year of obedience to God’s call without much visible evidence of God’s blessing?
 
-But before we go praising Noah and his steadfastness lets also remember the faithfulness of God. I know that in the areas of my life where I have been waiting on God the longest He has shown Himself faithful to me again and again, just as my faith was faltering and my hope was beginning to die God would remind me of His faithfulness to me and restore my hope. I am sure God did this with Noah too. On the days when Noah doubted (and I’m sure he had those days) God must have been there to remind him – just like He is there to remind us – not to give up.
+But before we go praising Noah and his steadfastness lets also remember the faithfulness of God. I know that in the areas of my life where I have been waiting on God the longest He has shown Himself faithful to me again and again. Just as my faith was faltering and my hope was beginning to die, God would remind me of His faithfulness to me and restore my hope. I am sure God did this with Noah too. On the days when Noah doubted (and I’m sure he had those days) God must have been there to remind him – just like He is there to remind us – not to give up.
```

Comment:
Yep :)

Sent from my iPhone

> On Jan 15, 2016, at 9:12 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> > @@ -51,7 +51,7 @@ Optional: if you didn’t already do so on Day 1 you can jot down a timeline of
> >  
> >  Reflect: As you read these verses today, try to imagine what it must have been like for Noah and his family to obey God day after day after day with no dramatic intervention or sign of blessing. That sounds a lot like my life too. While I may not have heard the audible voice of God or been assigned to build an ark, there are definitely areas in which I’ve been called to obedience and honestly, it can be hard for me to wait to receive the blessing of my obedience. I’ve fasted from something for a month and it has been eye-opening to see just how immediate-reward-focused I am! I am only on week two of my fast and I’m already itching to be done, questioning if this is actually doing me any good. What must it have been like for Noah as he logged year after year after year of obedience to God’s call without much visible evidence of God’s blessing?
> >  
> > -But before we go praising Noah and his steadfastness lets also remember the faithfulness of God. I know that in the areas of my life where I have been waiting on God the longest He has shown Himself faithful to me again and again, just as my faith was faltering and my hope was beginning to die God would remind me of His faithfulness to me and restore my hope. I am sure God did this with Noah too. On the days when Noah doubted (and I’m sure he had those days) God must have been there to remind him – just like He is there to remind us – not to give up.
> > +But before we go praising Noah and his steadfastness lets also remember the faithfulness of God. I know that in the areas of my life where I have been waiting on God the longest He has shown Himself faithful to me again and again. Just as my faith was faltering and my hope was beginning to die, God would remind me of His faithfulness to me and restore my hope. I am sure God did this with Noah too. On the days when Noah doubted (and I’m sure he had those days) God must have been there to remind him – just like He is there to remind us – not to give up.
> just breaking up run-ons
> 
> —
> Reply to this email directly or view it on GitHub.
>